change/@ni-nimble-components-e81dea16-5dd0-4d2d-8814-c5f263d310f1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
I likely need a refresher on why this change doesn't impact the built bits. Wouldn't changing our build tool versions potentially modify the build output?

And the facts that this is 
1. causing some tests to fail
2. causing chromatic diffs
3. changing the version of Angular packages that we test against 
...also give me pause. 

I guess our `package.json`s already said that clients could satisfy Nimble's dependencies with these newer versions. We just never tested against them and now that we are, we're discovering that they behave differently? Does that imply that we should have been pinning our package.json dependencies?

(possible we should have a call to talk through these points)

Reply:
In this case one of the changes was an update to `@tanstack/table-core` which had a behavior change around sorting undefined values. That's now resolved (and our tests did catch it) but since we're not only updating devDependencies, I'd say build output is definitely affected so these should probably be `patch` at least.

Reply:
Decision after offline discussion was to bump the tanstack version in package.json to reflect this change, which means this definitely merits a `patch`